src/index.js

```diff
@@ -195,7 +195,12 @@ export default function (babel) {
       if (!tailPath) continue;
 
       if (tailPath.isExpressionStatement()) {
-        tailPath.replaceWith(getNewNode(tailPath.node.expression, tailPath));
+        const expression = tailPath.node.expression;
+        if (t.isAwaitExpression(expression)) {
```

Comment:
this will only work for implicitly returned `await`s, right? 

... I think that's desired... but I'm not sure. thoughts?

might be helpful to add a comment here in any case